crates/unleash-edge-types/src/lib.rs

```diff
@@ -60,6 +61,12 @@ pub fn entity_tag_to_header_value(etag: EntityTag) -> HeaderValue {
 
 pub type BackgroundTask = Pin<Box<dyn Future<Output = ()> + Send>>;
 
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
I'm like 95% sure we'll need Copy here. This is tiny as well so it should be a non issue. If we have Copy I don't think we need Clone

```suggestion
#[derive(Debug, PartialEq, Eq, Copy)]
```